plugins/bulk-import/src/components/PreviewFile/PreviewPullRequestForm.tsx

```diff
@@ -68,6 +64,14 @@ export const PreviewPullRequestForm = ({
   const approvalTool =
     values.approvalTool === 'git' ? 'Pull request' : 'ServiceNow ticket';
 
+  const formik = useFormik<PullRequestPreviewData>({
+    enableReinitialize: true,
+    initialValues: pullRequest,
+    initialErrors: formErrors as any as FormikErrors<PullRequestPreviewData>,
```

Comment:
:eyes:

Why can the form has initial errors?

Reply:
This ensures that in the tabbed view, if there's an error in one tab, switching to another preview form within the same organization retains the errors and keeps the Save button disabled.

Reply:
I have updated the PR